src/parser.terms.js

```diff
@@ -0,0 +1,12 @@
+// This file was generated by lezer-generator. You probably shouldn't edit it.
+export const
+  FeatureModel = 1,
+  FeatureName = 2,
+  String = 3,
+  FeatureList = 4,
+  Feature3 = 5,
```

Comment:
?

Reply:
parser.tokens.js gets generated to enable indent and dedent but is  removed from repo